### DISCUSSION TITLE
Forking around

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#503] [Dev notes]
Just a quick tip, for those of you who are Forking this repository, the easiest way to keep your Fork `main` and feature branches up-to-date is to use the built-in **Sync fork, Update branch** feature in GitHub:

![69CB771F-23E7-4F12-AFA8-1929B671F72E](https://user-images.githubusercontent.com/8432125/222880643-1d4ebe25-2cf0-48f7-ad41-d9bd965bb8f2.jpeg)

**Comment by DaveSkender** [comment]
If you are contributing to this library, I recommend that you never commit changes to your Fork's `main` branch; rather, keep that up-to-date with the fork syncing feature shown above, then create a new feature branch from it for your incremental contribution.  When ready, create a Pull Request from your feature branch to the upstream source `main` branch (this repo, not to your Fork).

I've found this tutorial from @bmorelli25 helpful a while back when getting started with GitHub open-source contributions:
[A Step by Step Guide to Making Your First GitHub Contribution](https://codeburst.io/a-step-by-step-guide-to-making-your-first-github-contribution-5302260a2940)

**Reply by bitwizardry** [reply]
Cool!
